packages/contracts-frontend/README.md

```diff
@@ -0,0 +1,73 @@
+# @uma/contracts-frontend
```

Comment:
@Gamaranto @Tulun PTAL at this readme. This package should be strictly better than `@uma/core`. This package should be fully frontend compatible -- you should be able to `import { getVotingAbi, getVotingAddress, getVotingBytecode } from "@uma/contracts-frontend";` etc from a frontend.

The biggest change with this package is bundle size optimization. This package has 0 dependencies. The index file is over 2MBs because it contains all of the artifacts. However, unused functions get dropped from that file through webpack tree shaking. So, assuming you're only a few contracts, the package should end up taking up <10-20kb per contract factory that you import (according to my testing on large contracts, like Voting).

Another advantage is that you should now be able to pull in addresses by chain id from our repo: `getVotingAddress(chainId)`. This is something that I don't think you were able to do before.

The last nicety is that you can now import ethers or web3 contract types and factories a bit more easily:

```ts
import type { VotingEthers, VotingWeb3 } from "@uma/contracts-frontend";
import { VotingEthers__factory } from "@uma/contracts-frontend";
```

Reply:
Awesome work! The API is almost identical to the previous one we used and tree shaking was a much needed change!